pyuvdata/tests/test_parameter.py

```diff
@@ -42,6 +43,62 @@ def test_array_equality_nans():
     assert param1 == param2
 
 
+@pytest.mark.parametrize("atol", [0.001, 1 * units.mm])
+def test_quantity_equality(atol):
```

Comment:
I think this is a great test, but I think we should also try using values that are not exactly equal, but within the tolerance. (E.g., send in 1001 mm or 2999 mm). Then for completeness/paranoia we can use values that are just outside of the tolerance, and make sure those get flagged as not equal.

Reply:
good idea